views/account.erb

```diff
@@ -0,0 +1,46 @@
+
+<div class="container">
+  <h1>Account Information</h1>
+  <h2 class="tagline"><%=current_user.email%></h2>
+  <a href="/password">Change your password</a>
+  <% if current_user.premium %>
+    <h4 class="vip">Premium Account</h4>
+  <% else %>
+    <h4 class="vip">Basic Account</h4>
+  <% end %>
+</div>
+
+
+<div class="container">
+  <div class="main">
+
+  </div><!-- .main -->
+
+  <div class="sidebar">
+    <div class="widget">
+      <img src="https://www.gravatar.com/avatar/<%= Digest::MD5.hexdigest(current_user.email) %>" alt="Profile Picture" width="150">
+      <h2 class="profile">Stats</h2>
+    </div><!-- .widget -->
+    <div class="clear"></div>
+  </div><!-- .sidebar -->
+
+  <div class="widget" id="songs">
+    <h3>Suggested Songs</h3>
+    <ul>
+    <% current_user.suggested_songs.find_each do |song| %>
+      <li><p><strong>Title:</strong> <%= song.title %>, <strong>Artist:</strong> <%= song.artist %></p></li></br>
```

Comment:
You shouldn't generally need a `<br>` tag. If you want a break between list items, just make sure they are set to `display: block`